src/xo-app/menu/index.js

```diff
@@ -37,6 +37,7 @@ import styles from './index.css'
     [ task => task.status === 'pending' ]
   ),
   pools: createGetObjectsOfType('pool'),
+  hosts: createGetObjectsOfType('host'),
```

Comment:
If you just to know the number of hosts: `createGetObjectsOfType('host').count()`.